formats/hmrc_manual/publisher/links.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "topics": {
+      "anyOf": [
+        { "$ref": "#/definitions/guid_list" },
+        { "type": "null" }
+      ]
```

Comment:
We should never be sending `nil` in this field.  It should either be an array of GUIDs , or it should be not set at all.